review04/flight-service/src/test/java/cz/muni/fi/airportmanager/flightservice/repository/FlightRepositoryTest.java

```diff
@@ -0,0 +1,145 @@
+package cz.muni.fi.airportmanager.flightservice.repository;
+
+import cz.muni.fi.airportmanager.flightservice.entity.Flight;
+import cz.muni.fi.airportmanager.flightservice.model.FlightStatus;
+import io.quarkus.test.TestReactiveTransaction;
+import io.quarkus.test.junit.QuarkusTest;
+import io.quarkus.test.vertx.UniAsserter;
+import jakarta.inject.Inject;
+import org.junit.jupiter.api.Test;
+
+import java.time.Duration;
+import java.util.Date;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+
+@QuarkusTest
+class FlightRepositoryTest {
+
+    @Inject
+    FlightRepository flightRepository;
+
+    Flight createOngoingFlight() {
+        var future = Date.from(java.time.Instant.now().plus(Duration.ofMinutes(1000*60)));
+        var past = Date.from(java.time.Instant.now().minus(Duration.ofMinutes(1000*60)));
+        var flight = new Flight();
+        flight.setName("Test Flight");
+        flight.setAirportFrom("Airport A");
+        flight.setAirportTo("Airport B");
+        flight.setDepartureTime(past);
+        flight.setArrivalTime(future);
+        flight.setCapacity(100);
+        flight.setStatus(FlightStatus.ACTIVE);
+
+        return flight;
+    }
+
+    @Test
+    @TestReactiveTransaction
+    void shouldGetOngoingFlights(UniAsserter asserter) {
+        var flight = createOngoingFlight();
+        var currentDate = new Date();
+
+        asserter.execute(this.flightRepository::deleteAll)
+                .assertEquals(this.flightRepository::count, 0L)
+                .execute(() -> this.flightRepository.persist(flight))
+                .assertEquals(this.flightRepository::count, 1L)
+                .assertThat(
+                        () -> this.flightRepository.findOngoingFlights(currentDate),
+                        flights -> {
+                            assertEquals(1, flights.size());
+                            flight.setId(flights.get(0).getId());
+                            assertEquals(flight, flights.get(0));
+                        }
+                );
+    }
+
+    @Test
+    @TestReactiveTransaction
```

Comment:
This can probably go on the class so it doesn't need to be on every method

Reply:
I've created an issue to resolve this outside this MR